ovs/extensions/healthcheck/expose_to_cli.py

```diff
@@ -536,11 +569,16 @@ def get_command(self, ctx, name):
         :return: Function pointer to the command or None when no import could happen
         :rtype: callable
         """
+        cmd = self.commands.get(name)
```

Comment:
```
cmd = self.commands.get(name)
    if cmd: return cmd 
```
 returns quite a couple of times. 
Have you thought about ofloading it to a seperate function?

Reply:
Offloading won't really benefit it here

```
def get_possible_cached_command():
     return self.commands.get(name)

cmd = self.get_possible_cached_command()
if cmd:
  ....
```